src/account/AccountBalance.js

```diff
@@ -32,6 +37,7 @@ export default class AccountBalance {
      * @param {object} props
      * @param {Hbar} props.hbars
      * @param {?TokenBalanceMap} props.tokens
+     * @param {?TokenDecimalMap} props.tokenDecimals
```

Comment:
Wouldn't it be better to combine the balance map with the decimals? If someone wanted to get the decimals and the balances with two maps they'd need to do two null checks when in reality only one should be needed.